packages/experimental/Avatar/src/Avatar.tsx

```diff
@@ -1,132 +1,102 @@
+export type AvatarTokens = {
   /**
-   * The primary text to create initials with (e.g. a name)
+   * Supported Avatar sizes
    */
-  primaryText?: string;
+  size?: Size;
 
   /**
-   * The secondary text to create initials with if primary text is not provided (e.g. an email address)
+   * Supported Avatar styles
    */
-  secondaryText?: string;
+  avatarStyle?: AvatarStyle;
+};
 
+export type AvatarProps = AvatarTokens & {
   /**
    * The image to be displayed
    */
   imageSource?: ImageURISource;
 
   /**
-   * The color that represents this avatar.
-   * This color will override the initials view's background color.
-   * If the avatar view is configured to display a border, this will be the border's color.
-   * The colored border will not be displayed if a custom border image is provided.
+   * The primary text to create initials with (e.g. a name)
    */
-  color?: ColorValue;
+  primaryText?: string;
 
   /**
-   * Image to be used as border around the avatar. It will be used as a pattern image color,
-   * but It will be scaled to fit the avatar size. If set, the hasBorder initializer value will be ignored,
-   * since it's assumed that the client intends to have a custom border.
+   * The secondary text to create initials with if primary text is not provided (e.g. an email address)
    */
-  customBorderImageSource?: ImageURISource;
+  secondaryText?: string;
 
   /**
-   * The avatar view's presence state.
-   * The presence state is only shown when the style is set to AvatarStyle.circle.
+   * A custom color for the ring around the Avatar
    */
-  presence?: Presence;
-};
+  ringColor?: ColorValue;
 
-export type otherAvatarProps = {
   /**
-   * Supported Avatar sizes
+   * A custom color for the avatar initials when no image is shown
    */
-  size?: Size;
+  foregroundColor?: ColorValue;
 
   /**
-   * Background Color of initials
+   * A custom color for the avatar background when no image is shown
    */
   backgroundColor?: ColorValue;
 
   /**
-   * Image to be used as border around the avatar. It will be used as a pattern image color,
-   * but It will be scaled to fit the avatar size. If set, the hasBorder initializer value will be ignored,
-   * since it's assumed that the client intends to have a custom border.
-   */
-  customBorderImageSource?: ImageURISource;
-
-  /**
-   * Shape of the AvatarView
-   * Circle is used to represent people
-   * Square is used to represent organizations or teams
+   * The avatar view's presence state.
    */
-  avatarStyle?: AvatarStyle;
+  presence?: Presence;
 
   /**
-   * The color of the border of the avatar view
+   * A boolean inidicating whether the ring is visible
    */
-  borderColor?: ColorValue;
+  isRingVisible?: boolean;
 
   /**
-   * When true, the presence status border is opaque. Otherwise, it is transparent.
+   * A boolean indicating whether the Inner Ring Gap is transparent
    */
-  usesOpaquePresenceBorder?: boolean;
+  isTransparent?: boolean;
```

Comment:
Pretty misleading name here if it only applies to the inner ring gap and not the Avatar control as a whole being transparent

Reply:
The prop name comes from the FluentUI Apple control and maps 1:1. I can't really change that.

Reply:
This one applies on only to the ring inner gap, but also to the outer gap of the ring along with the presence icon (if available) and its outline.